playground/src/pages/basics/CollisionDemo.vue

```diff
@@ -0,0 +1,88 @@
+<script setup lang="ts">
+import { OrbitControls } from '@tresjs/cientos'
+import { TresCanvas } from '@tresjs/core'
+import { BallCollider, Physics, RigidBody } from '@tresjs/rapier'
+import { ACESFilmicToneMapping, SRGBColorSpace } from 'three'
+import { shallowRef } from 'vue'
+
+const gl = {
+  clearColor: '#82DBC5',
+  shadows: true,
+  alpha: false,
+  outputColorSpace: SRGBColorSpace,
+  toneMapping: ACESFilmicToneMapping,
+}
+
+const rigidTorusRef = shallowRef()
+const ballRef = shallowRef()
+
+const jump = () => {
+  if (!rigidTorusRef.value) { return }
+  rigidTorusRef.value.instance.applyImpulse({ x: 0, y: 5, z: 0 }, true)
+}
+const jumpBall = () => {
+  if (!ballRef.value) { return }
+  ballRef.value.instance.applyImpulse({ x: 0, y: 5, z: 0 }, true)
+}
+
+const onCollisionEnter = (event: any) => {
+  // eslint-disable-next-line no-console
+  console.log('jaime ~ ENTER', event)
+}
+const onCollisionEnterBall = (event: any) => {
+  // eslint-disable-next-line no-console
+  console.log('jaime ~ ENTER', event.source.rapierGroup.collider.collisionGroups())
+}
+const onCollisionExit = (event: any) => {
+  // eslint-disable-next-line no-console
+  console.log('jaime ~ EXIT', event)
+}
+</script>
+
+<template>
+  <TresCanvas v-bind="gl" window-size>
+    <TresPerspectiveCamera :position="[0, 11, 30]" :look-at="[0, 0, 0]" />
+    <OrbitControls />
+
+    <Suspense>
+      <Physics debug>
+        <RigidBody
+          ref="rigidTorusRef"
+          name="rigidTorus"
+          enableCcd
+          activeCollision
+          @collision-enter="onCollisionEnter"
+          @collision-exit="onCollisionExit"
+        >
+          <TresMesh :position="[0, 8, 0]" @click="jump">
+            <TresTorusGeometry />
+            <TresMeshNormalMaterial />
+          </TresMesh>
+          <TresMesh :position="[-5, 7, 0]">
+            <TresSphereGeometry />
+            <TresMeshNormalMaterial />
+          </TresMesh>
+        </RigidBody>
+
+        <RigidBody
+          @collision-enter="onCollisionEnterBall"
+        >
+          <BallCollider
+            ref="ballRef"
+            activeCollision
+            :args="[1, 1, 1]"
+            :position="[8, 15, 0]"
+            @click="jumpBall"
```

Comment:
I see you added a click event here, but colliders click is not implemented

Reply:
We should check this, I will remove the @click, but it's not working on custom-colliders (maybe is a bug)

Reply:
Are we supposed to bind click events to the physics components (Rigid body & Collider)?

Because it's not something that was implemented

Reply:
unmm IDK what would be the expected behavior for the user... something to have in mind

Reply:
In that case, we have to dig in and see it something that we have to implement 

We can create an issue about it

Reply:
Done: https://github.com/Tresjs/rapier/issues/136

As always, feel free to add more description to the issue

Reply:
Will do